native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp

```diff
@@ -34,8 +36,29 @@ static HMODULE libgles = NULL;
 static PFNGLES3WLOADPROC pfnGles3wLoad = NULL;
 
 bool gles3wOpen() {
-    libegl = LoadLibraryA("libEGL.dll");
-    libgles = LoadLibraryA("libGLESv2.dll");
+    std::string eglPath = "libEGL.dll";
+    std::string glesPath = "libGLESv2.dll";
+
+    #if CC_EDITOR
+    // In editor,there are same library,so we need to use abs path to load them.
+    char dllPath[MAX_PATH];
```

Comment:
MAX_PATH is 256 bytes, will user's dll path longer than that?

> Return value
> If the function succeeds, the return value is the length of the string that is copied to the buffer, in characters, not including the terminating null character. If the buffer is too small to hold the module name, the string is truncated to nSize characters including the terminating null character, the function returns nSize, and the function sets the last error to ERROR_INSUFFICIENT_BUFFER.

I think we need use Windows's GetLastError API to check whether we get the whole file path. If there is not, change a larger buffer and invoke it again until we succeed.

Reply:
fixed